<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoundsControlPanel</class>
 <widget class="QWidget" name="SoundsControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelVolume">
       <property name="font">
        <font>
         <family>Yuppy SC</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>设置音量</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="volumeSpinBox">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="volumeSlider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="repeatLayout">
     <item>
      <widget class="QLabel" name="repeatLabel">
       <property name="font">
        <font>
         <family>Yuppy SC</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>设置闹铃重复次数</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="repeatSpinBox">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QComboBox" name="soundsSelectBox">
     <property name="font">
      <font>
       <family>Yuppy SC</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
